contributors/devel/development.md

```diff
@@ -284,17 +284,31 @@ make cross
 ```sh
 cd $working_dir/kubernetes
 
+# Run all hack/update scripts
+# For faster deployment, run only relevant scripts:
+# hack/update-gofmt.sh (when adding new files)
+# hack/update-bazel.sh (when updating dependencies / imports)
+hack/update-all.sh
```

Comment:
yeah, sometimes you only want to update some.. I personally usually run `make verify` locally, and then run the relevant `hack/update-*` script to update the things that fail. Certain things like api changes though, it's easier to just run through everything with `make update`.